### ISSUE TITLE
Don't require _app.tsx and global stylesheet

### ISSUE THREAD

**Issue by shuding** [shuding/nextra#40]
Maybe we can remove step 4: 

![image](https://user-images.githubusercontent.com/3676859/101513662-66b70180-39b7-11eb-9602-7c7d6a45c1d1.png)

by just scope everything and use CSS Modules, including Tailwind (but under a `:global` scope ?)

**Comment by YikSanChan** [comment]
`pages/_app.js` is  where I add rss and google analytics support, not sure if removing this affects what I am doing?

```javascript
import "nextra-theme-blog/style.css"
import Head from "next/head"

export default function Nextra({ Component, pageProps }) {
  return (
    <>
      <Head>
        <link
          rel="alternate"
          type="application/rss+xml"
          title="RSS"
          href="/rss.xml"
        />
        <script
          async
          src="https://www.googletagmanager.com/gtag/js?id=[TRACKING_ID]"
        />
        <script
          dangerouslySetInnerHTML={{
            __html: `
            window.dataLayer = window.dataLayer || [];
            function gtag(){dataLayer.push(arguments);}
            gtag('js', new Date());
            gtag('config', '[TRACKING_ID]');
          `,
          }}
        />
      </Head>
      <Component {...pageProps} />
    </>
  )
}
```

**Comment by shuding** [comment]
Nope it won't affect that @YikSanChan. The current setup requires _app.js because it needs to inject the global style. With this feature the style will be automatically injected without _app.js. :-)

**Comment by YikSanChan** [comment]
@shuding Does that mean, there will still be the _app.js, just that we don't have to `import "nextra-theme-blog/style.css"` anymore?

**Comment by shuding** [comment]
@YikSanChan yep, you can still have _app.js if you want to customize it, but `import "nextra-theme-blog/style.css"` won't be needed anymore.

**Comment by hectorqin** [comment]
When will this feature come out? I want to merge two diffenrent app in one next.js, one is docs and another is a dashboard,but the global style has interrupt the dashboard style.

**Comment by B2o5T** [comment]
Released in `2.0.0-beta.14`
https://github.com/shuding/nextra/releases/tag/nextra-theme-docs%402.0.0-beta.14
https://github.com/shuding/nextra/releases/tag/nextra-theme-blog%402.0.0-beta.14
https://github.com/shuding/nextra/releases/tag/nextra%402.0.0-beta.14
